database/psdb.js

```diff
@@ -0,0 +1,49 @@
+const { Client } = require('pg');
+const credentials = require('../credentials');
+
+const client = new Client({
+  user: credentials.username,
+  host: credentials.host,
+  database: credentials.database,
+  password: credentials.password,
+  port: credentials.port
+});
+
+let currentid = 1;
```

Comment:
why are you hard coding a property id instead of passing one in from the server?